browser-test/src/support/admin_questions.ts

```diff
@@ -46,6 +51,10 @@ export class AdminQuestions {
     // expect(await this.page.isDisabled(`text=${questionName}`)).toEqual(true)
   }
 
+  async expectExportState(exportOption: string) {
+    expect(await this.page.isChecked(`label:has-text("${exportOption}") input`)).toEqual(true)
```

Comment:
Not quite sure I understand what's going on here. CI appears to be failing, even though this test passes when I run `bin/run-browser-tests` or `bin/run-browser-tests-ci`. Perhaps it could be something different about the underlying browser used by CI versus my local machine.

Reply:
Moving this back to draft mode while I try and figure out why CI is failing.

Reply:
This is now ready for review. Sorry about that!